### DISCUSSION TITLE
Group images in single line

### DISCUSSION THREAD

**Discussion by zaynetro** [fleather-editor/fleather#62] [Ideas]
Hi!

How would I implement grouping images together? I would like to display multiple images on a single row. Like:

![image](https://user-images.githubusercontent.com/627197/206707713-03333f59-f404-4163-918c-9f0c9bc43366.png)

And when user hits a backspace I want to either ignore the action or delete a single image.

Solutions that would work for me: 

* Support multiple block embeds per row
* Group images into a single block and allow disabling deleting embeds from the editor.
    Instead delete embeds manually through controller. (e.g when user presses a delete button)
    
Is there interest in support this feature? I will be glad to do a PR if you give me some guidance. 

P.S. As a workaround I guess I can currently listen to which blocks are removed and if multiple images are removed I reinsert the remaining ones.

**Comment by amantoux** [comment]
Hi @zaynetro ,

It is an interesting feature indeed. 
You may want to leverage on this [PR](https://github.com/fleather-editor/fleather/pull/16) to achieve this by implementing a new inline embed.

Happy to talk a look at your PR
